Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3565,3 +3627,270 @@ def remove_tag_from_endpoints_command(client: CoreClient, args: Dict):
     return CommandResults(
         readable_output=f'Successfully removed tag {tag} from endpoint(s) {endpoint_ids}', raw_response=raw_response
     )
+
+
+def parse_risky_users_or_hosts(user_or_host_data: dict[str, Any], table_headers: list[str]) -> dict[str, Any]:
```

Comment:
instead of passing a list of `table_headers`, pass three arguments: `id_header`, `score_header`, `description_header`.